core/consensus/babe/impl/babe_impl.hpp

```diff
@@ -109,9 +111,12 @@ namespace kagome::consensus {
     /**
      * Finish the Babe epoch
      */
-    void finishEpoch();
+    void nextEpoch();
```

Comment:
Hm, nextEpoch() reads like a getter, but it actually returns nothing and performs some logic instead. This is slightly confusing.

Reply:
Agree. Will be renamed to `startNextEpoch`